gnome-initial-setup/pages/endless-eula/gis-endless-eula-page.c

```diff
@@ -398,14 +109,14 @@ gis_endless_eula_page_constructed (GObject *object)
       g_error_free (error);
     }
 
-  load_css_overrides (page);
-
   gtk_widget_show (GTK_WIDGET (page));
-  load_terms_view (page);
 
   gis_page_set_forward_text (GIS_PAGE (page), _("_Accept and Continue"));
   gis_page_set_complete (GIS_PAGE (page), TRUE);
 
+  priv->eula_viewer = g_object_new (GIS_TYPE_ENDLESS_EULA_VIEWER, NULL);
```

Comment:
This needs to be unref'd in dispose/finalize

Reply:
It's going to be destroyed as part of the widget hierarchy as far as I can see, so the code we have here should be enough